pkg/distro/rhel/rhel8/distro.go

```diff
@@ -115,6 +115,15 @@ func newDistro(name string, minor int) *rhel.Distribution {
 			ImageFormat: platform.FORMAT_RAW,
 		},
 	}
+
+	// Keep the RHEL EC2 x86_64 images before 8.9 BIOS-only for backward compatibility.
+	// RHEL-internal EC2 images and RHEL AMI images are kept intentionally in sync
+	// with regard to not supporting hybrid boot mode before RHEL version X.Y.
```

Comment:
Nitpick: I'd replace `X.Y` with `8.9`, but I can live with it in the context of the first line.

Reply:
Thanks!